apps/examples/fs_helper/write_remove_test.c

```diff
@@ -53,17 +53,16 @@ static int make_file(char *filename)
 	}
 
 	for (i = 0; i < WRITE_DATA_SIZE; i++) {
-		ret = fwrite(&i, 4, 1, fp);
-		if (ret < 0) {
+		if (fwrite(&i, 4, 1, fp) < 0) {
 			printf("fwrite error[%d]\n", ret);
```

Comment:
printf will print previous / old value as fwrite error because it's not been set now .